dockerfiles/cross-build/Dockerfile.suse-15.1

```diff
@@ -7,12 +7,10 @@ ARG CREATE_USER="build"
 ARG USER_OPTIONS=""
 ENV PATH /go/bin:/usr/local/go/bin:$PATH
 
-RUN zypper install -y bash git make grep \
-    sed findutils coreutils \
-    tar bzip2 wget \
-    clang gcc \
-    kernel-devel \
-    rpm-build
+RUN zypper install -y rpm-build \
```

Comment:
15.2 was released. we might want to copy that dockerfile as 15.2 as well.

Reply:
Ok, just let me check it can build with that.